### PR TITLE
Refactor: Move to yarn workspace

### PR DESCRIPTION
**Pull request by imtsuki**
## Checklist

- [x] Move to yarn workspace
- [ ] Bundle @hedgehog/core separately
- [x] Fix CI scripts
- [x] Update docs

## Hierarchy

```
.
├── CONTRIBUTING.md
├── LICENSE
├── README.md
├── lerna.json
├── package.json
├── packages
│   ├── hedgehog-core
│   │   ├── package.json
│   │   ├── src
│   │   └── tsconfig.json
│   └── hedgehog-lab
│       ├── package.json
│       ├── public
│       ├── src
│       └── tsconfig.json
├── tsconfig.json
└── yarn.lock
```
Signed-off-by: imtsuki <me@qjx.app>

### CODE REVIEW

**Comment by AGDholo** [comment]
I pull this PR, and delete the `CI=true`, then my local project could work nice, 
But the console told me that `Project is running at http://192.168.123.205/`, 
How should I view the project on `localhost:port`?

**Comment by imtsuki** [comment]
@AGDholo There is a bug with `react-scripts` and `lerna` that the startup process will hang forever. `CI=true` is required to bypass this bug. I pushed a commit that uses `cross-env` to set the env var.

**Comment by AGDholo** [comment]
I noticed that github actions build error:
`FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory`

Should we add the flag command? `--max_old_space_size=8192`

File: packages/hedgehog-lab/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "cross-env": "^7.0.2",
+    "customize-cra": "^1.0.0",
+    "react-app-rewired": "^2.1.6",
+    "react-refresh": "^0.8.3",
+    "react-scripts": "3.4.1"
+  },
+  "scripts": {
+    "start": "cross-env CI=true react-app-rewired start",
+    "watch": "cross-env CI=true react-app-rewired start",
+    "build": "cross-env CI=true react-app-rewired build",
```
Comment (AGDholo) [inline packages/hedgehog-lab/package.json:36]: I noticed that github actions build error:
`FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory`

Should we add the flag command? `--max_old_space_size=8192`
Or create a github actions build script.

And 
Is source-map turned off in the default build situation? Do we need additional settings?

**Pull request merged**